FWI_forward_adjoint/FWIpart1.md

```diff
@@ -56,65 +56,68 @@ create highly optimized low-level C code. For this purpose Devito uses
 the symbolic algebra package SymPy [@Meurer17] to facilitate the
 automatic creation of derivative expressions, allowing the quick and
 efficient generation of high-order wave propagators with variable
-stencil orders.
+stencil orders, while automated performance optimization using
+state-of-the-art compiler techniques are used to ensure
+computational efficiency.
 
 At the core of Devito's symbolic API are two symbolic types that
 behave like SymPy function objects, while also managing
 user data:
 
-* `DenseData` objects represent a spatially varying function
+* `Function` objects represent a spatially varying function
   discretized on a regular cartesian grid. For example, a function
-  symbol `f = DenseData(name='f', shape=(nx, ny), space_order=2)`
+  symbol `f = Function(name='f', ..., space_order=2)`
```

Comment:
what's `...`

Reply:
The `grid`, but it's not relevant to talk about at this stage.

Reply:
I know, I just find it a bit confusing to avoid inputs not saying why or what it is. Maybe say it will be explained in further section and put back declaration with grid?